backend/app/api/v1/jobs.py

```
@@ -0,0 +1,224 @@
+    if await db.get(Application, application_id) is None:
+        raise HTTPException(status_code=404, detail="Application not found")
+    return list(
+        (await db.execute(select(InterviewScorecard).where(InterviewScorecard.application_id == application_id).order_by(InterviewScorecard.created_at.desc())))
+        .scalars()
+        .all()
+    )
+
+
+@router.get("/{job_id}/applicants/export/csv", response_class=StreamingResponse)
+async def export_job_applicants_csv(job_id: uuid.UUID, db: AsyncSession = Depends(get_db)) -> StreamingResponse:
+    job = await db.get(JobOpening, job_id)
```

Ruff (0.16.3)

[error] Undefined name `JobOpening`

(F821)